README.md

```diff
@@ -100,6 +100,8 @@ git checkout <latest tagged release>
 cargo build --release
 ```
 
+Note that compilation can require several GiB of memory. Please make sure that the machine has that amount of physical memory or swap has enough capacity (keep in mind that if builds hit swap it tends to be very slow).
```

Comment:
You should have said that more than 4GB  are required :P 

But people will not read this anyway and will continue to open prs xD

Reply:
Ah, 4GiB is also what I had in the back of my mind. I'll file another PR 

> But people will not read this anyway and will continue to open issues xD

Well, maybe, but at least those who reads this can save themselves a few back and forths